libbeat/docs/processors-list.asciidoc

```diff
@@ -86,6 +86,9 @@ endif::[]
 ifndef::no_include_fields_processor[]
 * <<include-fields,`include_fields`>>
 endif::[]
+ifndef::no_include_rate_limit_processor[]
+* <<rate-limit,`rate_limit`>>
```

Comment:
This line just adds a link. To fix the doc build error, you also need to add the include in the correct location under `//# tag::processors-include[]`

Reply:
Specifically, you want to add this to the correct location (alphabetically) in the list of includes:

```
ifndef::no_include_rate_limit_processor[]
include::{libbeat-processors-dir}/ratelimit/docs/rate_limit.asciidoc[]
endif::[]
```

Reply:
Thanks, fixed in a31b5ac53.